app/views/translation/database-tables/import.vue

```diff
@@ -0,0 +1,311 @@
+                                        </table>
+                                    </div>
+
+                                </div>
+                            </div>
+                    </div>
+
+                    <div class="row" v-show="logs.length">
+                            
+                            <div class="col-md-12">
+                                <div class="box box-default">
+                                    <div class="box-header with-border">
+                                        <h3 class="box-title">Logs</h3>
+                                    </div>                                    
+                                    <div class="box-body">
+                                        <div class="list-container logs">
+                                            <virtual-list class="stream scroll-touch" ref="vsl"
+                                            :data-key="'uid'"
+                                            :data-sources="logs"
+                                            :data-component="itemComponent"
+                                            :item-class="'stream-item'"
+                                            :keeps="logs.length"
+                                            :estimate-size="logs.length"
+                                            >
+                                            </virtual-list>                                                    
+                                        </div>
+                                    </div>
+
+                                </div>
+                            </div>
+                    </div>
+                </div>
+
+            </div>
+        </div>
+    </section>
+</template>
+
+<script>
+
+import VirtualList from 'vue-virtual-scroll-list'
+import simpleUpload from 'vue-simple-uploader'
+import   Multiselect         from 'vue-multiselect'
+import LogItem from './log-item.vue'
+
+const sleep = (ms)=>new Promise((resolve)=>setTimeout(resolve, ms));
+
+export default {
+    components: {
+        Uploader         : simpleUpload.Uploader,
+        UploaderBtn      : simpleUpload.UploaderBtn,
+        UploaderDrop     : simpleUpload.UploaderDrop,
+        UploaderUnsupport: simpleUpload.UploaderUnsupport,
+        UploaderList     : simpleUpload.UploaderList,
+        UploaderFiles    : simpleUpload.UploaderFiles,
+        UploaderFile     : simpleUpload. UploaderFile,
+        Multiselect,
+        VirtualList
+    },
+    data () {
+      return {
+        options: {
+            autoStart:false,
+            target: (file,request,c)=>{
+                const urls = {
+                    'application/zip'  : `/translation-api/database-table/${this.$route.params.table}/import/zip`,
+                    'application/json' : `/translation-api/database-table/${this.$route.params.table}/import/json`
+                }
+                let url = urls[file.fileType];
+                url = `${url}/${this.fileLanguage?.code}`;
```

Comment:
no encodeURIcomponent 
using ?. assum that the `fileLanguage` canc be `null` that will lead
to
`url/undefined`

Reply:
fixed